hypothesis-python/src/hypothesis/core.py

```diff
@@ -788,7 +788,7 @@ def run(data):
                                     test.__name__,
                                     args,
                                     kwargs,
-                                    force_split=True,
+                                    force_split=None,
```

Comment:
This makes verbose formatting similar to what is shown here: https://github.com/HypothesisWorks/hypothesis/blob/master/HypothesisWorks.github.io/_posts/2016-04-16-anatomy-of-a-test.md

I think that is much nicer, but I assume there's a reason it was `True` so feel free to say "no"...

Reply:
I think it's nice when the _whole call_ fits on a single line, but much worse when all the arguments are on a single line after the test name, and that turns out to be common in practice.  One-line-per-argument is a consistent way to get out of that, and also means that explain mode (#3555) looks similar to standard reports.